framework/src/kernels/EigenKernel.C

```diff
@@ -57,7 +80,7 @@ EigenKernel::computeResidual()
   _local_re.zero();
 
   Real one_over_eigen = 1.0;
-  one_over_eigen /= *_eigenvalue;
+  one_over_eigen /= (*_eigenvalue);
```

Comment:
I still think this should be 1 line instead of 2.  There's no more or less danger of divide-by-zero...